auction-server/src/auction/service/auction_manager.rs

```diff
@@ -522,10 +562,46 @@ impl Service<Svm> {
             relayer.sign_message(&serialized_message);
     }
 
+    #[tracing::instrument(skip_all, fields(bid_id, total_retries, tx_hash))]
+    async fn blocking_send_transaction(&self, bid: entities::Bid<Svm>, signature: Signature) {
+        tracing::Span::current().record("bid_id", bid.id.to_string());
+        tracing::Span::current().record("tx_hash", signature.to_string());
+        let config = RpcSendTransactionConfig {
+            skip_preflight: true,
+            max_retries: Some(0),
+            ..RpcSendTransactionConfig::default()
+        };
+        let mut receiver = self.config.chain_config.log_sender.subscribe();
+        for retry_count in 0..SEND_TRANSACTION_RETRY_COUNT_SVM {
+            tokio::time::sleep(Duration::from_secs(2)).await;
+
+            // Do not wait for the logs to be received
+            // just check if the transaction is in the logs already
+            while let Ok(log) = receiver.try_recv() {
+                if log.value.signature.eq(&signature.to_string()) {
+                    tracing::Span::current().record("total_tries", retry_count + 1);
```

Comment:
This is called `total_tires` but the field on top of the function is named `total_retries`.

Can we have this field in case of complete failure too? I know that we can assume if the span doesn't have this field, it means it retried the max amount but I guess it's just easier the other way.

Reply:
Good catch